### PR TITLE
ENH: adding a warning and correcting a few typo/grammar issue

### PR DESCRIPTION
**Pull request by awalter-bnl**
This PR is a result of a review of the deployment docs section. It involves adding a warning based on an issue from the last cycle, a few grammar re-arrangments and correction of a few typos. I also updated the name of the local anaconda server to alexandria.

### CODE REVIEW

File: source/deployment/resyncing-conda.rst
Comment (mrakitin) [inline source/deployment/resyncing-conda.rst]: First of all, something happened with the lines here, which made it wrong instructions. Second, I don't think we use the anaconda-server user on the machine. @licode, can you confirm?

Comment (awalter-bnl) [reply source/deployment/resyncing-conda.rst]: Thanks @mrakitin, these few lines where not part of the code-block which is why I attempted to fix them. It appears that I didn't arrange them correctly however, I have made the change.

**Comment by awalter-bnl** [comment]
The force push was done to resolve a git commit issue (2 almost identical commits ended up in the history, I squashed them together and then force pushed the result)

File: source/components/ansible-setup.rst
Comment (danielballan) [inline source/components/ansible-setup.rst]: It's worth considering changing `note` to `warning` and removing the shout-y "BE CAREFUL:"

**Review by danielballan — APPROVED** [review]
Thanks for these detail-oriented edits. I left one suggestion for your consideration.

File: .travis.yml
Comment (licode) [inline .travis.yml]: remove the line of python build.py ?

Comment (mrakitin) [reply .travis.yml]: @licode, it's the expected behavior in this case. The old installation method, which is used in v1.2.1 (released back in October 2018) relied on `python build.py` to build the shared libraries. However, in November they updated the building approach, and now everything is done with `pip install .` (see https://github.com/tomopy/tomopy/commit/21632927c3c78a5e3a4e3e01f67e4ef5b04a3edd).

I tried this new approach in #97, and it worked fine. Unfortunately, I didn't mention to keep it unmerged until after there is a newer tag of tomopy, so that we can use a tagged version as we did here in this PR. Anyway, this PR can be rebased on master with resolution of the conflicts to incorporate the changes from #97. For now we can continue using tomopy from the master branch.

**Comment by mrakitin** [comment]
Ups, GitHub's "Resolve conflicts" interface produced a merge commit. Will resolve it tomorrow morning.

**Comment by tacaswell** [comment]
`source/deployment/resyncing-conda.rst` <- that whole file in now wrong, @licode how do we do it now?

File: source/components/ansible-setup.rst
Comment (awalter-bnl) [reply source/components/ansible-setup.rst]: I have replaced this, although personally I think the shout-y "BE CAREFUL" is more likely to be noticed than a 'warning'.

**Comment by mrakitin** [comment]
Sorry I screwed up the history of the PR after force pushes last night. Now there is a merge-commit 49122949a7001832618495f805a03fa710b1f0fb, which could have been avoided if `git pull` was used before committing your changes. @danielballan, what's the best way to remove it?

**Comment by awalter-bnl** [comment]
@mrakitin  I did do a `git pull` prior to sending in the commit this morning, that is what added merge-commit 4912294. I can squash the last two together to resolve that, I will do so now.

**Comment by awalter-bnl** [comment]
OK so git rebase won't let me squash the merge commit, sorry.

**Comment by mrakitin** [comment]
No worries, thanks for trying.

**Comment by licode** [comment]
source/deployment/resyncing-conda.rst file is out of date. I can send pr to work on it. Do I send a new one after this get merged? Or I just push to the current pr? It looks like there were some issue, but just got resolved here.

**Comment by awalter-bnl** [comment]
@licode please just push to this one, that way we keep everything together.

File: source/deployment/resyncing-conda.rst
Comment (mrakitin) [inline source/deployment/resyncing-conda.rst]: I have a minor recommendation to use double-backticks or quotes surrounding `defaults` to make it distinguishable.

File: source/deployment/resyncing-conda.rst
Comment (mrakitin) [inline source/deployment/resyncing-conda.rst]: the script name can also be in quotes or backticks.

File: source/deployment/resyncing-conda.rst
Comment (mrakitin) [inline source/deployment/resyncing-conda.rst]: Sphinx has some functionality for the links, maybe it can be used for long links like this one?
See an example here: https://github.com/NSLS-II/docs/blob/85093efe8c5e7133a6af9a507f9bb82f7ecd1d7b/source/_cookbook/live_recon.py#L11

File: source/deployment/resyncing-conda.rst
Comment (mrakitin) [inline source/deployment/resyncing-conda.rst]: `conda_index` here and on the next row can also be surrounded by backticks or quotes for better readability.

File: source/deployment/resyncing-conda.rst
Comment (mrakitin) [inline source/deployment/resyncing-conda.rst]: "found at"

**Review by mrakitin — COMMENTED** [review]
Thanks @licode! I left a few comments to improve the readability. Please consider implementing them.

**Comment by licode** [comment]
thanks for suggestion!

File: source/deployment/resyncing-conda.rst
Comment (awalter-bnl) [inline source/deployment/resyncing-conda.rst]: I am having a hard time following this paragraph from line 23 to 28.
Am I correct in saying that line 19-21 indicate the commands required to synchronise packages with the latest anaconda versions?

I think lines 23 to 26 then should say something like: `

To modify which packages to synchronise modify the script found in `bootstrap-tag-build <https://github.com/NSLS-II/lightsource2-recipes/blob/master/scripts/bootstrapping/bootstrap-tag-build>` and
`nsls2-tag-build-local <https://github.com/NSLS-II/lightsource2-recipes/blob/master/scripts/nsls2-tag-build-local.sh>`_.
Note: A Token needs to be provided in the script (included a link to where the token can be found).

Is this what is meant?

For lines 26-27 I think you are implying that we need to run the `bash /opt/builds/run_conda_index` command each time? is this correct. if so I would make this a note.

File: source/deployment/resyncing-conda.rst
Comment (awalter-bnl) [inline source/deployment/resyncing-conda.rst]: I think this should be written as `Moreover a sync of ``lightsource2-tag`` to ``nsls2-tag`` follows a similar process:`

File: source/deployment/resyncing-conda.rst
Comment (awalter-bnl) [inline source/deployment/resyncing-conda.rst]: should be `is running on ``alexandria``.....`

File: source/deployment/resyncing-conda.rst
Comment (awalter-bnl) [inline source/deployment/resyncing-conda.rst]: should be: `included in a cronjob every 6 mins`

**Review by awalter-bnl — COMMENTED** [review]
Thanks for adding this @licode : 

There is one section here that is still a little confusing to me. I also included a few other grammar changes that make my OCD kick in !-)

File: source/deployment/resyncing-conda.rst
Comment (licode) [reply source/deployment/resyncing-conda.rst]: correct. working on it.

Comment (licode) [reply source/deployment/resyncing-conda.rst]: thanks for those comments.

**Comment by licode** [comment]
Done.

**Review by awalter-bnl — COMMENTED** [review]
This now looks good to me, thanks @licode and @mrakitin for the help.

File: source/components/ansible-setup.rst
Comment (danielballan) [inline source/components/ansible-setup.rst:67]: This needs a blank line. The `git submodule ...` line is being interpreted as a setting on `code-block` rather than content of the block.

From the Travis build:

```
/home/travis/build/NSLS-II/docs/source/components/ansible-setup.rst:66:Error in "code-block" directive:
maximum 1 argument(s) allowed, 6 supplied.
```

File: source/components/ansible-setup.rst
Comment (danielballan) [inline source/components/ansible-setup.rst:62]: I'm not sure that a blank line is _required_ here but I suggest leaving one for consistency with directives that do require them.

File: source/components/ansible-setup.rst
Comment (danielballan) [inline source/components/ansible-setup.rst:71]: Same here. FWIW I'm not sure these need to be offset as notes -- maybe just more prose? -- but I leave that up to you.

File: source/components/ansible-setup.rst
Comment (awalter-bnl) [reply source/components/ansible-setup.rst:67]: done

File: source/components/ansible-setup.rst
Comment (awalter-bnl) [reply source/components/ansible-setup.rst:62]: I have added it, like @danielballan I prefer to be consistent even if it isn't necesary

File: source/components/ansible-setup.rst
Comment (awalter-bnl) [reply source/components/ansible-setup.rst:71]: I prefer to leave them as notes, but not strongly if others are adamant they need to be changed let me know and I will change them

**Pull request merged**
